Index: src/ovasCAcquisitionServerGUI.cpp
===================================================================
--- src/ovasCAcquisitionServerGUI.cpp	(Revision 3009)
+++ src/ovasCAcquisitionServerGUI.cpp	(Arbeitskopie)
@@ -9,6 +9,7 @@
 #include "brainproducts-brainampseries/ovasCDriverBrainProductsBrainampSeries.h"
 #include "brainproducts-brainvisionrecorder/ovasCDriverBrainProductsBrainVisionRecorder.h"
 #include "brainproducts-vamp/ovasCDriverBrainProductsVAmp.h"
+#include "contec-kt88/ovasCDriverContecKT88.h"
 #include "ctfvsm-meg/ovasCDriverCtfVsmMeg.h"
 #include "egi-ampserver/ovasCDriverEGIAmpServer.h"
 #include "emotiv-epoc/ovasCDriverEmotivEPOC.h"
@@ -127,6 +128,7 @@
 	m_vDriver.push_back(new CDriverBrainProductsVAmp(m_pAcquisitionServer->getDriverContext()));
 #endif
 	if(l_bShowUnstable) m_vDriver.push_back(new CDriverCtfVsmMeg(m_pAcquisitionServer->getDriverContext()));
+	m_vDriver.push_back(new CDriverContecKT88(m_pAcquisitionServer->getDriverContext()));
 	if(l_bShowUnstable) m_vDriver.push_back(new CDriverEGIAmpServer(m_pAcquisitionServer->getDriverContext()));
 #if defined TARGET_HAS_ThirdPartyEmotivAPI
 	m_vDriver.push_back(new CDriverEmotivEPOC(m_pAcquisitionServer->getDriverContext()));